setup.py

```diff
@@ -16,8 +16,11 @@
     classifiers=[
         "Environment :: Web Environment",
         "Framework :: Django",
+        "Framework :: Django :: 3.2",
+        "Framework :: Django :: 4.0",
```

Comment:
**Question**: If we are supporting Wagtail >=2.15 then Django==4.0 would be a problem I think.